src/analysis/calculus/deriv.lean

```diff
@@ -902,6 +902,10 @@ has_deriv_at_filter.tendsto_nhds inf_le_left h
 theorem has_deriv_at.continuous_at (h : has_deriv_at f f' x) : continuous_at f x :=
 has_deriv_at_filter.tendsto_nhds (le_refl _) h
 
+theorem has_deriv_at.continuous_on (h : ∀ x ∈ s, has_deriv_at f f' x) :
+  continuous_on f s :=
+λ x hx, continuous_at.continuous_within_at (has_deriv_at.continuous_at (h x hx))
```

Comment:
These lemmas are designed to be chained together, i.e.
```suggestion
λ x hx, (h x hx).continuous_at.continuous_within_at
```

Reply:
Update:  I see that it was adding this lemma which broke the build.  That's because, whenever someone has done `open has_deriv_at`, the predicate `continuous_on` now gets confused with the lemma `has_deriv_at.continuous_on`.

This could be fixed by going through all the broken files and changing the occurences of `continuous_on` to `_root_.continuous_on`.  Or you could change the name of this lemma to something unambiguous like `continuous_on_of_has_deriv_at`.

Or you could just delete the lemma, since it doesn't save *that* many characters -- compare
```
  have hcu' : continuous_on u _ := has_deriv_at.continuous_on hu,
```
with
```
  have hcu' : continuous_on u _ := λ x hx, (hu x hx).continuous_at.continuous_within_at,
```